modules/common/src/test/scala/surge/internal/health/matchers/RepeatingSignalMatcherSpec.scala

```diff
@@ -63,8 +63,8 @@ class RepeatingSignalMatcherSpec extends TestKit(ActorSystem("RepeatingSignals")
         Range(1, 100),
         Range(1, 100))
 
-      Source(repeatingData)
-        .mapAsync(parallelism = 10)(data =>
+      Source(repeatingData.asInstanceOf[immutable.Iterable[Range]])
```

Comment:
Cleaning up a warning here I'm guessing? Maybe following the pattern in the Akka Streams docs for [`Source.from`](https://doc.akka.io/docs/akka/current/stream/operators/Source/from.html) would be better here? Even though this is just tests, `asInstanceOf` makes me nervous in general